airflow/providers/google/cloud/example_dags/example_dlp.py

```diff
@@ -177,3 +178,31 @@
     )
     # [END howto_operator_dlp_delete_job_trigger]
     create_trigger >> update_trigger >> delete_trigger
+
+DEIDENTIFY_CONFIG = {
```

Comment:
Would it be worth also including the config in the documentation right before the example? Given the fact that this config isn't just a string, I think it might be worth showing

Reply:
Good shout - I've added some more general info as as well as a snippet of the de-identify config